core/src/main/java/org/mifos/mobilewallet/core/data/fineract/api/FineractApiManager.java

```diff
@@ -37,12 +37,12 @@
 
     public static final String DEFAULT = "default";
     public static final String BASIC = "Basic ";
-    private static BaseURL baseUrl = new BaseURL();
+        private static BaseURL baseUrl = new BaseURL();
```

Comment:
Were these changes required to make the build pass on latest changes? They seem odd to me.

Reply:
Ah, it is just to test if the workflow runs as expected, I would revert these changes